src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, inject } from '@angular/core';
+import { Breakpoints, BreakpointObserver } from '@angular/cdk/layout';
+import { map } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css']
+})
+export class DashboardComponent {
+  private breakpointObserver = inject(BreakpointObserver);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BreakpointObserver` should be injected via the constructor, not with the `inject` function.
```diff
- private breakpointObserver = inject(BreakpointObserver);
+ constructor(private breakpointObserver: BreakpointObserver) {}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->